base_search_fuzzy/tests/test_query_generation.py

```diff
@@ -37,8 +36,8 @@ def test_fuzzy_where_generation(self):
             where_clause_params)
         self.assertEqual(
             complete_where,
-            'SELECT FROM "res_partner" WHERE '
-            '("res_partner"."name" % \'test\')')
+            b'SELECT FROM "res_partner" WHERE '
```

Comment:
I think the `b` (binary) is not needed

Reply:
It's necessary because complet_where is binary.

Reply:
Uhm, I don't think so. @Yajo what do you think?

Reply:
I have no reasons to believe he's wrong, and Travis was green, so it's OK for me.